google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java

```diff
@@ -1137,6 +1125,37 @@ public void close() throws Exception {
     return optionMap;
   }
 
+  static Map<Option.OptionType, ?> updateFilter(final Map<Option.OptionType, ?> options) {
+    String existingFilter = EntryListOption.OptionType.FILTER.get(options);
```

Comment:
nit: some function comments would be helpful

Reply:
Added